src/app/book/components/book/book.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input, Output, EventEmitter } from '@angular/core';
+import { BookModel } from '../../../core/models/book';
+
+@Component({
+  selector: 'app-book',
+  templateUrl: './book.component.html',
+  styleUrls: ['./book.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BookComponent implements OnInit {
+  @Input() book!: BookModel;
+  @Output() bookOrder = new EventEmitter<BookModel>();
+
+  constructor() { }
+
+  ngOnInit(): void {
+    console.log(this.book);
```

Comment:
same
suppose, you don't need console.log anymore

Reply:
fixed